docs/samples/management-api-v2-walkthrough/1-management-api-oveview.md

```diff
@@ -0,0 +1,67 @@
+# Management API V2 Overview
+
+## Introduction
+
+With the introduction of the new [Dataspace Protocol](https://docs.internationaldataspaces.org/dataspace-protocol/overview/readme), now using JSON-LD, all Management API endpoints had to be adapted as well to reflect that.
+JSON-LD (JSON for Linked Data) is an extension of JSON that introduces a set of principles and mechanisms to enable interoperability.
+
+This document will showcase how this change impacts the management API usage.
+
+> Please note: Before running the examples the corresponding environment variables must be set.
+> How such an environment can be setup locally is documented in [settings changes](../../migration/Version_0.3.4_0.4.0.md).
+
+## 1. Modified Endpoints
+
+The `MANAGEMENT_URL` specifies the URL of the management API and the prefix `v2` allows access to the functionality of the new management API.
+
+| Resource              | Endpoint                                   |
+|-----------------------|--------------------------------------------|
+| Asset                 | `<MANAGEMENT_URL>/v2/assets`               |
+| Policy Definition     | `<MANAGEMENT_URL>/v2/policydefinitions`    |
+| Contract Definition   | `<MANAGEMENT_URL>/v2/contractdefinitions`  |
+| Catalog               | `<MANAGEMENT_URL>/v2/catalog`              |
+| Contract Negotiation  | `<MANAGEMENT_URL>/v2/contractnegotiations` |
+| Contract Agreement    | `<MANAGEMENT_URL>/v2/contractagreements`   |
+| Transfer Process      | `<MANAGEMENT_URL>/v2/transferprocesses`    |
+
+> Please note: The old Management API is now deprecated and is not tested for compliance. The `/v2` prefix is only a temporary part of the path and will be discarded once the migration to the new protocol is finished and the old API is taken out of service. Once that is done, the new management API can be reached through `MANAGEMENT_URL`.
```

Comment:
With the EDC Connector release v0.1.0 and therefore also TX v0.4.1 (note that v0.4.0 still has the deprecated API) the old management API was taken out of service already.

Reply:
As the note is intended to warn about the usage of the `/v2` prefix (and it's still there), I think its worth keeping it.